### DISCUSSION TITLE
Merging reports for GitHub Actions

### DISCUSSION THREAD

**Discussion by CommanderTvis** [cqfn/diKTat#1434] [Q&A] [answered]
Is it possible to merge Diktat reports from Gradle subprojects in order to get a single SARIF file and upload it with GitHub Actions?

**Comment by orchestr7** [comment, marked as answer]
Yeah, of course - we have an example of action for unifying and publishing SARIF reports here: 
https://github.com/saveourtool/save-cloud/blob/master/.github/workflows/diktat.yml

Check this one also: https://github.com/akuleshov7/reflekt/blob/master/.github/workflows/diktat.yml

We are doing the same for detekt and other reporting tools. 

PS If that answers your question pls press "mark as answer"
